### PR TITLE
* Some testing

### PR DESCRIPTION
**Pull request by Wagnerp**
* Some testing for #1281 

image

**Note:** Cannot currently define a user input to use, will be fixed in a future PR

### CODE REVIEW

File: Source/Krypton Components/TestForm/UserInputToastNotificationTest.cs [hunk 1]
```diff
@@ -1,12 +1,274 @@
-﻿using Krypton.Toolkit;
+﻿using System;
```
Comment (Smurf-IV) [inline Source/Krypton Components/TestForm/UserInputToastNotificationTest.cs:1]: Why are these here ?

Comment (Smurf-IV) [reply Source/Krypton Components/TestForm/UserInputToastNotificationTest.cs:1]: Also These new files needs to have the License header added.

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

**Comment by Wagnerp** [comment]
Will review on Thursday, as I'm currently away until Wednesday

File: Source/Krypton Components/TestForm/UserInputToastNotificationTest.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/TestForm/UserInputToastNotificationTest.cs]: Sorted

File: Source/Krypton Components/TestForm/BasicToastNotificationTest.cs [hunk 2]
```diff
@@ -1,4 +1,16 @@
-﻿namespace TestForm
+﻿#region BSD License
+/*
+ * 
+ * Original BSD 3-Clause License (https://github.com/ComponentFactory/Krypton/blob/master/LICENSE)
```
Comment (Smurf-IV) [inline Source/Krypton Components/TestForm/BasicToastNotificationTest.cs]: Does not need the `ComponentFactory` part, as these are all new

**Review by Smurf-IV — CHANGES_REQUESTED** [review]
